gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/GatewayHomepageController.java

```diff
@@ -34,6 +34,9 @@ public class GatewayHomepageController {
 
     private static final String SUCCESS_ICON_NAME = "success";
     private static final String WARNING_ICON_NAME = "warning";
+    protected static final String ERROR_TITLE = "The API Mediation Layer does not work";
+    protected static final String WARNING_TITLE = "There are limitations to how API Mediation Layer works";
+    protected static final String SUCCESS_TITLE = "The API Mediation Layer works properly";
```

Comment:
I think there's no need to bring changes going to the old gateway, in any case we should make sure these changes are moved to the SCGW. cc @pj892031 @achmelo